example/generator/odp_generator.c

```diff
@@ -25,6 +25,8 @@
 #define MAX_WORKERS            (ODP_THREAD_COUNT_MAX - 1)
 #define POOL_NUM_PKT           2048  /* Number of packets in packet pool */
 #define POOL_PKT_LEN           1856  /* Max packet length */
```

Comment:
I do not see why POOL_PKT_LEN is still needed. Now it acts as minimum packet length for pool creation (not maximum as said in its comment) in case the length derived from command line parameters is very small. But
I do not see why there is a need to have such minimum instead of simply always deriving the packet length parameter from the command line parameters.

Reply:
Will change comment to "Default Pool Buffer Length", and pool creates with this value when either payload not passed or requested < 1500. This enables generator similar to before change, also no issues with receiver mode for regular MTU.